cirq-google/cirq_google/engine/engine_processor_test.py

```diff
@@ -124,6 +126,165 @@ def _to_timestamp(json_string):
 )
 
 
+_CIRCUIT = cirq.Circuit(
+    cirq.X(cirq.GridQubit(5, 2)) ** 0.5, cirq.measure(cirq.GridQubit(5, 2), key='result')
+)
+
+
+_RESULTS_V2 = _to_any(
```

Comment:
For clarity, could we make this (and the below proto) use a builder followed by a serialization instead of the raw serialization ?